mongo/filterstore.go

```diff
@@ -15,19 +16,22 @@ const Submitted = "submitted"
 
 var NotFound = errors.New("not found")
 var NotAuthorised = errors.New("Not authorised")
+var DimensionNotFound = errors.New("Dimension not found")
+var Forbidden = errors.New("Forbidden")
 
 // FilterStore containing all filter jobs stored in mongodb
 type FilterStore struct {
 	Session *mgo.Session
```

Comment:
Not part of the changes made here, but consider making the session private, and passing it in on the constructor